### ISSUE TITLE
December 18 Leadership Retreat

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/iced-coffee#198]
At the beginning of this year, we made a [Strategic Plan for 2016](https://docs.google.com/document/d/12EGMSZL2wGKpxwcxCFCTMoeRfx95h4eF493Bh5kdkkA/edit). A lot has changed in 10 1/2 months.  This "retreat" is designed as place where we will discuss openly the purpose and trajectory of our organization. We will invite new leaders to join the retreat. 

We will encourage honest discussion and document all decisions made and action items generated.

**This will happened at** Google Office (3300 N Interstate 35),
**On Sunday, December 18th** starting at 9am.
**There will be light coffee and lunch,**
**provided by** Google Fiber.

Here is a list of topics as a starting point (this will be updated frequently):
- Events
  - Hack Nights
  - Hackathons
  - Code Across, Skillshares, Design Workshops,etc
  - Library Meetups
  - Promotion
- Roles
  - Who does what? What do we call the person who does that thing that needs doing?
  - How are decisions made? How are leadership roles and responsibilities transferred over time?
- Projects
  - Are individual projects reaching their intended goals?
  - Onboarding, project management, marketing
- Advocacy
  - How do we influence policy? 
  - How does our membership participate in creating policy initiatives?
- Sustainability
  - How much money did we spend vs receive this year?
  - What do we need money for?
  - What other resources do we have/need and are we utilizing them well?
    - AWS credits (could be expiring)
    - in kind donations
- Structure
  - Should OA be we "just" a meetup group, "official" CfA brigade that can receive donations through a 3rd party (if that continues to exist), or a full-fledge non-profit org? Or something else entirely?
  - Legal status or no? Bank account or no? Bylaws? Articles of Incorporation? Lawyer fees?

**Comment by mateoclarke** [comment]
_Random Thoughts_
- Mission over Brand
    - We have done a lot this year to increase the value of our brand. I'm pointing to a new logo, new website, t-shirts & buttons, more social media activity, more blog posts, more presentations at community orgs.
  - What have we done to clarify/strengthen our mission?
- Events and online community as "public squares"
  - What is the value of publicly owned convening spaces?
- [platform coorporative](http://platformcoop.net/)
  - This is a new academic-y word. Think of an uber model where the drivers own shares of the software/business model.
  - is there an opporunity for us to have an online labor brokerage for civic tech projects? This already exisits for volunteer opportunities, does that work well?
- Decentralize leadership
  - Take power from a captain and move it outwards. Take power from the core team and move it outwards
- Does the movement keep moving without you?
  - thought experiment: What would happen to civic tech in Austin if we all stopped doing anything?

**Comment by danielryne** [comment]
**TL;DR** - I'm volunteering to facilitate and proposing an agenda/structure similar to a business model canvass. Let me know what you think (be 100% honest, won't hurt my feelings) and/or if you'd like to volunteer to help. 

I'd like to volunteer myself to help plan and facilitate this retreat, and welcome help from anyone who is interested in planning this as well. As a facilitator, my role would be to help structure and guide the conversation toward the goals set by the group, not necessarily contribute to the conversation - which will be best left to interested members and leaders of Open Austin. 

**Agenda Proposal 1 - Not my first choice**
There are clearly a lot of topics to cover and not enough time to dive thoroughly into each. That said, I think it will benefit everyone to first establish some common vision and goals for the organization. Here's a basic breakdown of what I think that could look like if we stick to one day: 

9:00 am - Coffee / Set up 
9:30 - Values Conversation - unlike other groups, values are at the forefront on conversation for OA. However, I still think it will be worthwhile to reaffirm the values that drive OA, as well as reflect on what that looked like in programming this past year. 
10:00 - Visioning for 2017 - Identifying some priority spaces and activities that OA wants to be a part of or take a lead in 2017 and beyond. 
11:00 - Goal setting - A quick overview of BMC, as well as identifying the things we want to take away from the retreat. 
11:30 - Lunch 
12:00 pm - BMC Session 1 - Based on visioning, discuss OA's key activities, value propositions, and channels. 
2:00 pm - Break 
2:15 pm - Organizational Development - Normally, this is where you'd take what you identified from the BMC and begin to identify key partners, resources and revenue streams, but OA already has much of that identified. So, I suggest we use this time to talk about organizational development - ie. team roles and organizational sustainability. 
4:30 pm - Assess / reaffirm activities, set homework and next steps 
5:00  pm- Clean up and done

**Agenda Proposal 2 - My first choice**
You'll notice above that I propose utilizing the [business model canvass](https://strategyzer.com/canvas/business-model-canvas) to help us flesh out the gritty details of the vision for 2017 and set some next steps. This requires that the group to get on the same page in terms of values and visioning, which takes up most of the morning.

Alternatively, if we wanted more time to discuss revenues streams and get into more detail about organizations structure and sustainability, OA could set aside some time at a general meeting do some pre-retreat visioning and goal setting. This would actually be my first choice, as it not only helps us identify things before the meeting, but also ensures that we've collected feedback from a wider audience.

Okay, I'll keep giving this some more thought but would love some feedback from folks!

**Comment by mateoclarke** [comment]
@danielryne, Thanks for proposing this agenda! And for providing two options.

We haven't scheduled a speaker for our Monday, November 14th library meetup. Also, Amalie and I will be going to Code for America Summit the first week in November. I had been thinking that the 11/14 meetup would be a good time to report back about what happened at Summit in regards to the brigade network. I also thought we could have a discussion about the state of civic tech with events like Sunlight Foundation Labs phasing out as one example. And I think this would all set up a pretty good conversation about values, visioning, and goal setting.

Would that work well for 2nd proposal?

**Comment by danielryne** [comment]
I think that would work well!

**Comment by amaliebarras** [comment]
Sounds great to me, good ideas y'all. So let's plan on the whole core team being present at that Nov. 14th meetup?

**Comment by VictoriaODell** [comment]
Meeting the 14th Sounds solid. 

I mentioned earlier on slack that I'll be out on Dec 4-8 so it would be great if we could pick another time for the retreat.

**Comment by mateoclarke** [comment]
We're looking at Sat, Dec 17th or Sun, Dec 18th as alternative dates. Can y'all chime if y'all have a preference for either of those days?

@danielryne @amaliebarras @VictoriaODell @mscarey @luqmaan @gusIreland @mapineda

**Comment by mapineda** [comment]
Will be in Winston-Salem for a wedding that weekend.

**Comment by amaliebarras** [comment]
no strong preference

**Comment by danielryne** [comment]
Any final thoughts on a date for this? I'd like to book the venue asap. 

If you're concerned about missing out, please know that the retreat is not the end all of the future direction of the organization. Quite the contrary, it's the beginning of the conversation. So there ill be opportunity to weigh in on things afterwards.

**Comment by VictoriaODell** [comment]
Any of those 17-18 dates work for me.

**Comment by mateoclarke** [comment]
Hey @danielryne, yeah let's go with either the 17th or 18th

**Comment by mscarey** [comment]
Yeah, I think either one is OK

**Comment by danielryne** [comment]
Great, let's shoot for the 18th then! I'll get us a room at the Google (3300 n interstate 35) for 9:00 am that Sunday.

**Comment by mateoclarke** [comment]
@danielryne, wanted to share these notes from an old core team huddle which I found relevant to how we describe the importance and value of the brigade:
https://github.com/open-austin/iced-coffee/issues/165

**Comment by danielryne** [comment]
Hey all, just wanted to update everyone! We're still on for this Sunday 12/18 from 9:00 am to 3:00 pm. The location will be 3300 N Interstate 35 (Google Office, 2nd Floor). There is free parking all around and in the parking garage attached to the office, so feel free to park anywhere. We're hoping to order Thunderbird Coffee for the morning and Thunderbird sandwiches for lunch. If he hasn't already, Mateo will be reaching out to you about orders for lunch! 

Below is the tentative agenda. Please let me know any questions or comments you have! 

-Danny
--------------------
9:00 am	Coffee & Intros 				
9:30 am 	Values Recap (Mateo & Vickie)
10:00 am Roles & Responsibilities 			
11:30 am Lunch & Google Tour (Optional)		 
12:30 pm Project Planning and Calendaring		
1:45 pm 	Break 
2:00 pm 	Membership and Leadership Development

**Comment by amaliebarras** [comment]
Thanks Danny for putting this together for us, the agenda looks great!

Do you have the assets from the value mapping activity we did? Could you bring those or upload pictures to the drive so we can reference them while we work?

**Comment by danielryne** [comment]
Hey Amalie, yes! I have them and didn't get around to transcribing them. But I can bring them. I also switched you as one of the co-leads on that session, fyi.

**Comment by amaliebarras** [comment]
No need to transcribe. And yes, thank you :) I knew what you meant

**Comment by danielryne** [comment]
Just so folks have my cell phone: 956-970-9214

If you're coming in from the parking garage, you'll need to take the elevators to the first floor and then hang a right, then take the main elevators to the second floor and go through the glass doors. Call or text if you get lost!

**Comment by danielryne** [comment]
Link to annotated agenda: https://docs.google.com/document/d/1oRZkISMorYLoID-4SbPA6sk6ZOHvXqGyeDJHXgVZD2U/edit?usp=sharing

**Comment by mateoclarke** [comment]
This was awesome. Huge thanks to @danielryne for facilitating this and getting the space and food provided by Google!

I created a Drive Folder where we can store photos and output from the Leadership Summit. I would like to schedule a follow up meeting in early Jan so we can regroup about some of the tasks ahead. But let's get through the holidays first. ⛄️ 

Thanks for everything y'all:
https://drive.google.com/open?id=1Q0uEKvycpmGalquGgkBawt8PAdueKYShMECIbjSEM00

**Comment by mateoclarke** [comment]
original pasted the wrong link, corrected.
https://drive.google.com/open?id=0B1cMlEZm_nJ0OG5Ta0lUSDd5RUU
